docs/development/documentation/index.mdx

```diff
@@ -1,19 +1,20 @@
 ---
-id: documentation
+id: docs
 title: Maintaining Documentation
+slug: /development/docs
 sidebar_label: Documentation
-sidebar_position: 6
+sidebar_position: 9
 ---
 
 Meshtastic documentation is an important ingredient to the overall project. We want users to hit the ground running with the information they need right at their finger tips. This section will discuss the documentation software stack, file organization, and style guides.
 
 ## Software Stack
 
-All of our documentation resides on GitHub. Instructions for setting up your GitHub account are located [here](github).
+All of our documentation resides on GitHub. Instructions for setting up your GitHub account are located [here](/docs/development/reference/github).
 
-Our documentation is powered by [Docusaurus](https://docusaurus.io) — a documentation platform built on React that utilizes markdown files. Because markdown files are easy to edit, most content changes should be fairly simple. Instructions for setting up your instance of Docusaurus are located [here](docusaurus).
+Our documentation is powered by [Docusaurus](https://docusaurus.io) — a documentation platform built on React that utilizes markdown files. Because markdown files are easy to edit, most content changes should be fairly simple.
 
-Another component that we use is [Vercel](https://vercel.com) — a platform for frontend frameworks and static sites. Instructions for setting up your instance of Vercel are located [here](vercel).
+Another component that we use is [Vercel](https://vercel.com) — a platform for frontend frameworks and static sites. Instructions for setting up your instance of Vercel are located [here](/docs/development/documentation/publish#publish-to-vercel).
 
 ## Documentation Organization
```

Comment:
As i mentioned on discord we might want to make some changes here to reflect this reorganizing of the docs.

Reply:
Agreed, I once I get one of our last hurdles out the way (protoc-gen-docs) we can have pr reviews built automatically